src/components/product-card-grid/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactElement } from 'react'
+import Link from 'next/link'
+import classNames from 'classnames'
+import { IconConsulColor16 } from '@hashicorp/flight-icons/svg-react/consul-color-16'
+import { IconTerraformColor16 } from '@hashicorp/flight-icons/svg-react/terraform-color-16'
+import { IconVaultColor16 } from '@hashicorp/flight-icons/svg-react/vault-color-16'
+import { IconNomadColor16 } from '@hashicorp/flight-icons/svg-react/nomad-color-16'
+import { IconPackerColor16 } from '@hashicorp/flight-icons/svg-react/packer-color-16'
+import { IconVagrantColor16 } from '@hashicorp/flight-icons/svg-react/vagrant-color-16'
+import { IconWaypointColor16 } from '@hashicorp/flight-icons/svg-react/waypoint-color-16'
+import { IconBoundaryColor16 } from '@hashicorp/flight-icons/svg-react/boundary-color-16'
+import { IconHashicorp16 } from '@hashicorp/flight-icons/svg-react/hashicorp-16'
+import s from './style.module.css'
+import CONTENT from './content.json'
+
+const logoDict = {
+  terraform: IconTerraformColor16,
+  vault: IconVaultColor16,
+  consul: IconConsulColor16,
+  nomad: IconNomadColor16,
+  packer: IconPackerColor16,
+  vagrant: IconVagrantColor16,
+  waypoint: IconWaypointColor16,
+  boundary: IconBoundaryColor16,
+}
+
+const iconDict = {
+  hcp: IconHashicorp16,
+}
```

Comment:
This is very similar to what `ProductIcon` does, are you able to use that component here?

Reply:
@ashleemboyer Ah, nice! Thanks for calling this out 💯  I should be able to, might need to tweak a couple things so I can pass `className`. Will give that a shot, and will add `ProductIcon` to swingset 👍

Reply:
@ashleemboyer Have updated to use `ProductIcon`, and added [some swingset docs for ProductIcon](https://dev-portal-git-zsscaffold-home-page-hashicorp.vercel.app/swingset/components/producticon) as well. Let me know what you think!